src/utils.js

```diff
@@ -136,7 +136,35 @@ export const isPruneTraceOff = () =>
   );
 
 export const isSwitchedOff = () => {
-  return TracerGlobals.getTracerInputs().switchOff;
+  return TracerGlobals.getTracerInputs().switchOff || !isValidAlias();
```

Comment:
I'm missing logs telling that aliases are in use

Reply:
if anything will be filtered by the aliases - there's an info log inside the filter function